kmwine02/project01
# Wesley branch

To Test:

- Open index.html
- Search "Elton John" and press "Search"
- Verify venues appear in artist cards
- Click "Find a Brewery"
- Verify you are redirected to brewery.html
